silhouette/test/com/mohiva/play/silhouette/impl/providers/BasicAuthProviderSpec.scala

```diff
@@ -135,15 +150,29 @@ class BasicAuthProviderSpec extends PlaySpecification with Mockito {
     lazy val fooHasher = {
       val h = mock[PasswordHasher]
       h.id returns "foo"
+      h.isSuitable(any) answers { p =>
```

Comment:
Yes, the default implementation doesn't get used.

```
p.asInstanceOf[PasswordInfo].hasher == "foo"
```

Is much better. I'll change it.